transaction-view/src/instructions_frame.rs

```diff
@@ -17,6 +17,15 @@ pub(crate) struct InstructionsFrame {
     pub(crate) num_instructions: u16,
     /// The offset to the first instruction in the transaction.
     pub(crate) offset: u16,
+    pub(crate) cached_offsets_and_lens: Vec<InstructionOffsetAndLens>,
```

Comment:
nit: to me, this is just another type of `Frame`, so I think it could be renamed to something like:

```suggestion
    /// Instruction framing data
    pub(crate) instructions: Vec<InstructionFrame>,
```